src/Web/HTML/Common.purs

```diff
@@ -0,0 +1,30 @@
+module Web.HTML.Common where
+
+import Prelude
+
+import Data.Newtype (class Newtype)
+
+-- | A type-safe wrapper for property names.
```

Comment:
I'm not sure calling these "type safe" is exactly right, maybe just "semantic"?

Reply:
Happy to remove it. Just copying what already exists in Halogen https://github.com/purescript-halogen/purescript-halogen/blame/c8b50378b948fcba9e10d75da718fc7dae3a609f/src/Halogen/HTML/Core.purs#L171